content/html/concepts/tags/terms/kbd/kbd.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '<kbd>'
+Description: 'Emphasizes characters to look like keys on a keyboard.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Style'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<kbd>` tag emphasizes characters to look like keys on a keyboard.
+
+## Syntax
+
+The `<kbd>` tag has no attributes. Text is placed in between the opening and closing `<kbd>` tags and will render in the browser's default monospace font. `<kbd>` can be nested but the result will be the same.
+
+```pseudo
+<kbd><!-- Text goes here --></kbd>
+```
+  
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>C</kbd>.</p>
+```
+
+Result:
+
+![Keyboard Tag Example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-1.png)
```

Comment:
I noticed a small typo with the filenames. It should say "*kbd*-tag-example-1.png" (emphasis mine), instead. We will need to rename both this file and the other, too.

To make this change, you have to make it locally by running the following command:

```
git mv kdb-tag-example-1.png kbd-tag-example-1.png
```

Then, run `git add` + `commit` + `push` back up to this branch.